tests/serializations.py

```diff
@@ -25,15 +25,19 @@
 import copy
```

Comment:
looks like Blake did a big job

Reply:
Glad you like it, it's increasing the diff and the blame depth but worth it imho